<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encapsulation::Internal::SettingsWidget</class>
 <widget class="QWidget" name="Encapsulation::Internal::SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="behaviorGroupBox">
     <property name="title">
      <string>Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="addGetCheckBox">
        <property name="text">
         <string>Add &quot;&amp;get&quot; to accessor name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mutatorFirstCheckBox">
        <property name="text">
         <string>Insert mutator &amp;first</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="camelCaseCheckBox">
        <property name="text">
         <string>Use camelCase</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cppFileCheckBox">
        <property name="text">
         <string>Insert definitions in .&amp;cpp file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fieldNamesGroupBox">
     <property name="title">
      <string>Field names</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="prefixEdit">
        <property name="text">
         <string>m_</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="suffixLabel">
        <property name="text">
         <string>&amp;Prefix:</string>
        </property>
        <property name="buddy">
         <cstring>prefixEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="suffixEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="prefixLabel">
        <property name="text">
         <string>&amp;Suffix:</string>
        </property>
        <property name="buddy">
         <cstring>suffixEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
